C-Sharp.md

```diff
@@ -1,6 +1,268 @@
-#The C# Programming Language
+# The C# Programming Language
 
+1. ### Basics
+  * Setup
+  * Your first C# program
```

Comment:
Not really. What I mean is this:

Your first C# program would be Your First C# Program